src/data/faq.json

```diff
@@ -1418,8 +1419,8 @@
                                 "anchor": "check_in_luca_warn_for_others",
                                 "active": false,
                                 "textblock": [
-                                    "<b>Updated January 18, 2023</b>",
-                                    "This feature of the Corona-Warn-App is no longer available from version 3.0 onwards and cannot be used with the previous versions 2.9 to 2.28 after January 31, 2023, as the verification hotline (TAN hotline) will be discontinued on this date.",
+                                    "<b>Updated February 1, 2023</b>",
+                                    "This feature of the Corona-Warn-App is no longer available since version 3.0 and cannot be used with the previous versions 2.9 to 2.28 after January 31, 2023, as the verification hotline (TAN hotline) will be discontinued on this date.",

```

Comment:
"will be" => "was"

Reply:
Updated in https://github.com/corona-warn-app/cwa-website/pull/3357/commits/b006857fcb5fdb081ce62f9f90250306044fa7f1, thanks for the suggestion!